python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -510,13 +510,27 @@ def queue_monitor_streaming(self) -> None:
     def management_api_http(self) -> None:
         """management_api_http set based on management_eapi data-model."""
         if self.inputs.management_eapi.enabled:
-            self.structured_config.management_api_http.enable_vrfs.append_new(name=self.inputs.mgmt_interface_vrf)
             self.structured_config.management_api_http._update(
                 enable_http=self.inputs.management_eapi.enable_http,
                 enable_https=self.inputs.management_eapi.enable_https,
                 default_services=self.inputs.management_eapi.default_services,
             )
 
+            # TODO: remove in AVD 6.0
```

Comment:
@Vibhu-gslab modified this way - can you please check that it is properly covered by tests?

Reply:
Added a negative TC

Reply:
note to other reviewer: we also have positive TCs for this all over the place